src/components/ProfileStepper.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
```

Comment:
This file shouldn't be a part of this PR

Reply:
Again, shouldn't be part of this PR

Reply:
`ProfileStepper` is not longer part of this PR